src/main/java/lotto/controller/LottoPages.java

```diff
@@ -22,20 +22,31 @@ public int getSize() {
         return pages.size();
     }
 
-    public int getPrizes(int[] winnerNumber) {
-        return pages.stream().mapToInt(page -> Arrays.stream(WinnerEnum.values())
+    public int getPrizes(int[] winnerNumber, int bonusNumber) {
+        return pages.stream().mapToInt(page -> Arrays.stream(PrizeEnum.values())
                 .filter(
-                        winnerEnum -> page.LottoCompare(winnerNumber) == winnerEnum.getMatch()).findAny()
-                .orElse(WinnerEnum.FAIL).getPrize()
+                        prizeEnum -> page.LottoCompare(winnerNumber, bonusNumber) == prizeEnum.getMatch()
+                ).findAny().orElse(PrizeEnum.FAIL).getPrize()
         ).reduce(0, Integer::sum);
     }
 
-    public int getPrizesByEnum(WinnerEnum winnerEnum, int[] winnerNumber) {
-        return (int) pages.stream().filter(page -> page.LottoCompare(winnerNumber) == winnerEnum.getMatch()).count();
+    public String getPrizesContentByEnum(PrizeEnum prizeEnum, int[] winnerNumber, int bonusNumber) {
+        if (prizeEnum == PrizeEnum.FAIL) return "";
```

Comment:
한 줄로 끝나는 실행문이라도 중괄호를 꼭 사용해주세요. 

```suggestion
        if(prizeEnum == PrizeEnum.FAIL) {
            return "";
        }
```

Reply:
반영했습니다